2.1/jessie/amd64/sdk/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/dotnet-nightly:2.1.300-preview1-jessie
```

Comment:
Base SDK tag used across platforms isn't consistent.  This particular usage isn't using the explicit SDK tag while others are.